devtools/server/actors/webbrowser.js

```diff
@@ -1316,7 +1316,12 @@ TabActor.prototype = {
       // In child processes, we have new root docshells,
       // let's watch them and all their child docshells.
       if (this._isRootDocShell(docShell)) {
-        this._progressListener.watch(docShell);
+        // Don't assume there's a progress listener, since Positron starts
+        // without a window, so this._progressListener never gets created.
+        // TODO: ensure there's a listener for apps without an initial window.
+        if (this._progressListener) {
```

Comment:
I spotted a few more cases where you might want the same thing:

https://dxr.mozilla.org/mozilla-central/rev/f3f2fa1d7eed5a8262f6401ef18ff8117a3ce43e/devtools/server/actors/webbrowser.js#1397
https://dxr.mozilla.org/mozilla-central/rev/f3f2fa1d7eed5a8262f6401ef18ff8117a3ce43e/devtools/server/actors/webbrowser.js#1478